__tests__/.eslintrc

```
@@ -0,0 +1,6 @@
+{
+  "plugins": ["jest"],
+  "env": {
+    "jest/globals": true
+  }
```

This'll handle loading the jest plugin + setting up globals for you:

```suggestion
  "extends": ["@readme/eslint-config/testing"],
```